rpcpb/rpc.proto

```diff
@@ -185,6 +189,7 @@ message ClusterInfo {
   // The map of blockchain IDs in "ids.ID" format to its blockchain information.
   map<string, CustomChainInfo> custom_chains = 8;
   repeated string subnets          = 9;
+  map<string, SubnetParticipants> subnetParticipants = 10;
```

Comment:
can we change the name to subnet_participants to follow same convention of others